modules/installation-configuration-parameters.adoc

```diff
@@ -220,6 +220,22 @@ ifdef::osp[]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.openstack.rootVolume.size`
+|For compute machines, the size in gigabytes of the root volume in the machine pool. If you do not set this value, machines use ephemeral storage.
```

Comment:
Not sure what "machine pool" refers to here. I'd leave "in the machine pool" out.

Reply:
@mandre Just repeating a term from the upstream docs. If it's superfluous, I'm fine removing it. :+1: